fe/fe-core/src/main/java/org/apache/doris/datasource/FileScanNode.java

```diff
@@ -61,6 +61,7 @@ public abstract class FileScanNode extends ExternalScanNode {
     // For explain
     protected long totalFileSize = 0;
     protected long totalPartitionNum = 0;
+    protected long tableLevelRowCount = -1;
```

Comment:
Looks like this field is only used for showing in `explain` result?
If yes, add some comment to it